src/Panel/Ui/Stat.php

```diff
@@ -90,6 +90,10 @@ public function props(): array
 
 	protected function stringTemplate(string|null $string = null): string|null
 	{
+		if ($this->model === null) {
+			return $string;
+		}
+
 		if ($string !== null) {
 			return $this->model->toString($string);
```

Comment:
Maybe simpler and with a single line?

```suggestion
			return $this->model?->toString($string) ?? $string;
```

Reply:
I first had it like that but then felt it was easier to read with its own if statement on top.

Reply:
OK fine 👍